applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -285,6 +289,9 @@ def GetDefaultParameters(cls):
             "fm_ale_settings": {
                 "fm_ale_step_frequency": 0,
                 "mesh_movement": "implicit",
+                "embedded_velocity_calculation": "from_fluid_mesh_velocity",
+                "rbf_interpolation_search_radius": 0.0,
```

Comment:
you could make this a multiplier to the max edge length, this way It could be almost fully automatic

Reply:
Better this than the bounding box :+1: . Thanks for the suggestion.

Reply:
Done.